<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GiveCashDialog</class>
 <widget class="QDialog" name="GiveCashDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Інформація про видачу</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBoxDialGiveCash">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDialGiveCash">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Видано готівку на суму :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Всього видано купюр :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>500 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>200 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_8">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>90</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_9">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>120</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_10">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>150</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_11">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>180</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_12">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>210</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1DialGiveCash_13">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>240</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDialGiveCash_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDialGiveCash_3">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxDialGiveCash</sender>
   <signal>accepted()</signal>
   <receiver>GiveCashDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxDialGiveCash</sender>
   <signal>rejected()</signal>
   <receiver>GiveCashDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
